docs/schemas/shared/IfcSharedBldgElements/Types/IfcSlabTypeEnum.md

```diff
@@ -32,7 +32,7 @@ The slab is used to represent a wearing surface.
 The slab is used to represent a sidewalk.
 
 ### TRACKSLAB
-A track slab is a reinforced concrete slab or prestressed reinforced concrete slab, which is a main element of slab track. It can be prefabricated or cast on site and may have sleepers embedded.
+A track slab is a reinforced concrete slab or prestressed reinforced concrete slab, which is a main element of a slab track. It can be prefabricated or cast on site and may have sleepers embedded.
```

Comment:
```suggestion
A reinforced concrete slab or prestressed reinforced concrete slab, which is a main element of a slab track. It can be prefabricated or cast on site and may have sleepers embedded.
```

^ shouldn't we remove name here as well?

Also I don't really understand this definition. "TRACKSLAB is the main element of a slab track?" Really? But I guess that's not something we can resolve now.